src/MSIdentityScaffolding/Microsoft.DotNet.MSIdentity/DeveloperCredentials/MsalTokenCredential.cs

```diff
@@ -120,21 +121,30 @@ public override async ValueTask<AccessToken> GetTokenAsync(TokenRequestContext r
             {
                 if (ex.Message.Contains("AADSTS70002")) // "The client does not exist or is not enabled for consumers"
```

Comment:
can we add a source link to this failure type/code in a comment above.

Reply:
Yeah I'll add this: https://learn.microsoft.com/en-us/azure/active-directory/develop/reference-aadsts-error-codes